pkgs/by-name/su/sudo/package.nix

```diff
@@ -72,7 +75,14 @@ stdenv.mkDerivation (finalAttrs: {
     rm $out/share/doc/sudo/ChangeLog
   '';
 
-  passthru.tests = { inherit (nixosTests) sudo; };
+  passthru = {
+    tests = { inherit (nixosTests) sudo; };
+    updateScript = genericUpdater {
+      versionLister = writeShellScript "sudo-versionLister" ''
+        ${lib.getExe curl} -sL https://www.sudo.ws/dist | grep -Po 'href="sudo-\K[\w.]*(?=\.tar\.gz")'
+      '';
+    };
```

Comment:
Is the generic `directoryListingUpdater` unsuitable for this case?

Reply:
Sadly it is; it filters out the versions that contain `p`.